docs/troubleshooting/harvester.md

```diff
@@ -67,6 +67,223 @@ Users can generate a support bundle in the Harvester GUI with the following step
     ![](/img/v1.2/troubleshooting/harvester-sb-support-modal.png)
 
+### Manually Download and Retain a Support Bundle File
+
+By default, a support bundle file is automatically generated, downloaded, and deleted after you click **Create** on the Harvester UI. However, you may want to retain a file for various reasons, including the following:
+
+- You are unable to download the file because of network connectivity errors and other issues.
+
+- You must use a previously generated file to troubleshoot issues (because generating a support bundle file takes time).
+
+- You want to view information that only exists in a previously generated file.
+
+Even if the file remains in the cluster, the Harvester UI does not provide a download link. Use the following workaround to generate, manually download, and retain a support bundle file:
+
+#### Generate the File and Prevent Automatic Downloading
+
+1. On the Harvester UI, click **Generate Support Bundle**.
+
+1. When the progress indicator reaches 20% to 80%, close the browser tab to prevent automatic downloading of the generated file.
+
+1. Retrieve a list of all support bundles in all namespaces using kubectl.
+
+    Example:
+    ```
+    $ kubectl get supportbundle -A
+    NAMESPACE          NAME           ISSUE_URL   DESCRIPTION   AGE
+    harvester-system   bundle-htl5f               sp1           3h43m
+    ```
+
+1. Retrieve the details of all existing support bundles using the command `kubectl get supportbundle -A -o yaml`.
+
+    Example:
+    ```
+    $ kubectl get supportbundle -A -oyaml
+    apiVersion: v1
+    items:
+    - apiVersion: harvesterhci.io/v1beta1
+      kind: SupportBundle
+      metadata:
+        creationTimestamp: "2024-02-02T11:18:09Z"
+        generation: 5
+        name: bundle-htl5f  // resource name
+        namespace: harvester-system
+        resourceVersion: "1218311"
+        uid: a3776373-05fe-4584-8a9a-baac3fa91bbf
+      spec:
+        description: sp1
+        issueURL: ""
+      status:
+        conditions:
+        - lastUpdateTime: "2024-02-02T11:18:38Z"
+          status: "True"
+          type: Initialized
+        filename: supportbundle_db25ccb6-b52a-4f9d-97dd-db2df2b004d4_2024-02-02T11-18-10Z.zip  // support bundle file name
+        filesize: 8868712
+        progress: 100  // 100 means successfully generated
+        state: ready
+    ```
+
+The file is ready for downloading when the value of `progress` is "100" and the value of `state` is "ready".
+
+#### Download the File
+
+1. Create a download URL that includes the following information:
+
+    - VIP or DNS name
+    - Resource name of the file
+    - Parameter `?retain=true`: If you do not include this parameter, resources related to the support bundle are automatically deleted after the file is successfully downloaded.
+
+    Example:
+
+    `https://{vip/dns-name}/v1/harvester/supportbundles/bundle-htl5f/download?retain=true`
```

Comment:
Just one question, should we add a notice to tell users this vip/dns-name is harvester dashboard ip/dns-name?

Reply:
L133 is updated as `[VIP](../install/management-address.md)`